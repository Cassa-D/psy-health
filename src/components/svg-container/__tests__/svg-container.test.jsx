import React from "react";
import renderer from "react-test-renderer";

import SvgContainer from "../index";

describe("Renders", () => {
  it("shows", () => {
    const tree = renderer.create(<SvgContainer svgExtra="" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
