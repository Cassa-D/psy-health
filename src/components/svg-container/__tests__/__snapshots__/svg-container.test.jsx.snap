// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders shows 1`] = `
<div
  className="sc-bdnylx sc-gtssRu gYbAMc"
>
  <img
    alt="Curva"
    src="curva.svg"
  />
  <div>
    <img
      alt="svgExtra"
      src=""
    />
  </div>
</div>
`;
